### PR TITLE
[Doc Improvement][Add code block for node.js][3621999]

### PR DESCRIPTION
**Pull request by v-npaladugu**

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/messaging-extensions/how-to/action-commands/respond-to-task-module-submit.md](https://github.com/MicrosoftDocs/msteams-docs/blob/c0ef68488acec3f94743b58d7d4778d6191a7e3f/msteams-platform%2Fmessaging-extensions%2Fhow-to%2Faction-commands%2Frespond-to-task-module-submit.md) | 90 | 12 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/36716729-de05-4b9a-b1ad-b7f0147f2a49) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/messaging-extensions/how-to/action-commands/respond-to-task-module-submit.md [hunk 1]
```diff
@@ -559,19 +559,32 @@ To use the user attribution in teams, you must add the `OnBehalfOf` mention enti
...
-      }  
-    }
+channelData: {
+    onBehalfOf: [ {
+        itemId: 0,
+            mentionType: 'person',
+            mri: context.activity.from.id,
+            displayName: context.activity.from.name
+    } ]
+}
```
Comment (Wajeed-msft) [inline msteams-platform/messaging-extensions/how-to/action-commands/respond-to-task-module-submit.md:569]:
```suggestion
// Attribute the message to the user on whose behalf the bot is posting
  responseActivity.ChannelData = new {
    OnBehalfOf = new []
    {
      new
      {
        ItemId = 0,
        MentionType = "person",
        Mri = turnContext.Activity.From.Id,
        DisplayName = turnContext.Activity.From.Name
      }  
    }
  };
```

File: msteams-platform/messaging-extensions/how-to/action-commands/respond-to-task-module-submit.md [hunk 2]
```diff
@@ -559,19 +559,32 @@ To use the user attribution in teams, you must add the `OnBehalfOf` mention enti
...
+```javascript
+channelData: {
+    onBehalfOf: [ {
+        itemId: 0,
+            mentionType: 'person',
+            mri: context.activity.from.id,
+            displayName: context.activity.from.name
+    } ]
+}
+
```
Comment (Wajeed-msft) [inline msteams-platform/messaging-extensions/how-to/action-commands/respond-to-task-module-submit.md:584]:
```suggestion
    const responseActivity = { type: 'message', attachments: [adaptiveCard], channelData: {
        onBehalfOf: [ { 
            itemId: 0, 
            mentionType: 'person', 
            mri: context.activity.from.id, 
            displayname: context.activity.from.name 
            }
        ]
    }};
```

**Review by Wajeed-msft — APPROVED** [review]

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/messaging-extensions/how-to/action-commands/respond-to-task-module-submit.md](https://github.com/MicrosoftDocs/msteams-docs/blob/6f5747acea496aa714d35231fd111fa8c9c9ddf8/msteams-platform%2Fmessaging-extensions%2Fhow-to%2Faction-commands%2Frespond-to-task-module-submit.md) | 90 | 12 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/9b3221fa-4fb6-4733-aac4-bfb019c2bd5e) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/messaging-extensions/how-to/action-commands/respond-to-task-module-submit.md](https://github.com/MicrosoftDocs/msteams-docs/blob/415d090868168c095cfb9511b98624b32f63e76e/msteams-platform%2Fmessaging-extensions%2Fhow-to%2Faction-commands%2Frespond-to-task-module-submit.md) | 90 | 12 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/dbf027b2-7903-444b-8a34-f2cce30a6776) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by Wajeed-msft — APPROVED** [review]

**Review by surbhigupta12 — APPROVED** [review]

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/messaging-extensions/how-to/action-commands/respond-to-task-module-submit.md](https://github.com/MicrosoftDocs/msteams-docs/blob/40e7cdeb1005a8376e2272cb109fc541e933bd9a/msteams-platform%2Fmessaging-extensions%2Fhow-to%2Faction-commands%2Frespond-to-task-module-submit.md) | 90 | 12 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/c0bb1cc5-1ad0-477e-a641-18d22838568a) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Pull request merged**